lib/cartopy/feature.py

```diff
@@ -185,6 +186,41 @@ def geometries(self):
 
         return iter(geometries)
 
+class GADMFeature(Feature):
+    """
+    A simple interface to GADM (version 2) shapefiles.
+
+    See http://www.gadm.org/
+
+    """
+    def __init__(self, name, level, **kwargs):
+        """
+        Args:
+
+        * name:
+            The name of the administrative area as a three letter
+            code, e.g. 'GBR' or 'USA'.
+        * level:
+            The 
```

Comment:
Unfinished sentence, though I assume that's because of the issues raised in the PR.